templates/dashboard/support.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'default.html' %}
+
+{% set active_page = None %}
```

Comment:
I think the `active_page` can be set to `dashboard`

Reply:
:+1: